### PR TITLE
develop error page and enhance nav bar

### PR DESCRIPTION
**Pull request by Wence17**
image

I enhanced the navbar responsiveness.

image

Yet to understand nextjs rendering so I have to alter the code to do the rendering.  But I set it to the default after testing my code.

### CODE REVIEW

File: app/components/Navbar.tsx [hunk 1]
```diff
@@ -4,28 +4,29 @@ import Image from "next/image"
 
 function Navbar() {
   return (
-    <div className="bg-white h-[50px] flex items-center justify-center ">
```
Comment (KMF21) [inline app/components/Navbar.tsx:7]: lets make the height (10px  mobile)(30px medium)(50px large)

**Review by KMF21 — CHANGES_REQUESTED** [review]

**Comment by KMF21** [comment]
well-done @Wence17

take note of this
errorw
cheers!!

File: app/components/Navbar.tsx (hunk 1 above)
Comment (Wence17) [reply app/components/Navbar.tsx:7]: This is not my last push

Comment (Wence17) [reply app/components/Navbar.tsx:7]: image




check this out.  Defined my screen sizes for the container class or it's not necessary?

File: app/page.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import Image from 'next/image'
 import { Inter } from 'next/font/google'
 import Navbar from './components/Navbar'
+import ErrorPage from './ErrorPage'
```
[the remark this answers is not in the document]
Comment (Wence17) [reply app/page.tsx:4]: Alright!

done

File: app/components/ErrorsCodes/Error404.tsx
[the remark this answers is not in the document]
Comment (Wence17) [reply app/components/ErrorsCodes/Error404.tsx:1]: image



check out this documentary.
https://nextjs.org/learn/seo/crawling-and-indexing/status-codes



also see this stackoverflow questions
https://stackoverflow.com/questions/70712833/nextjs-custom-error-page-with-correct-status-code
https://stackoverflow.com/questions/74991106/how-to-handle-status-codes-at-the-middleware-level

File: app/components/Navbar.tsx (hunk 1 above)
Comment (Wence17) [reply app/components/Navbar.tsx:7]: I did not add this to the new push, just a suggestion.

**Review by KMF21 — APPROVED** [review]

**Pull request merged**